### PR TITLE
make all functions top level

### LINKED ISSUE #55: Make all function top level
An issue to keep track of moving all function to be top level like in Treecle.

### PR DESCRIPTION
**Pull request by adamjanicki2**
Closes #55

### CODE REVIEW

**Comment by LeaVerou** [comment]
??

**Comment by adamjanicki2** [comment]
> ??

Now it's ready, realized I made bug fixes in this PR and wanted to do them in a separate one 😅

File: src/children.js [hunk 1]
```diff
@@ -1,6 +1,2 @@
 import "./treecle-setup.js";
-import { children, childPaths, replace } from "../lib/treecle/src/index.js";
-
-export { children as of, childPaths as paths, replace };
-export default children;
-
+export { children, childPaths, replace as replaceChild } from "../lib/treecle/src/index.js";
```
Comment (LeaVerou) [inline src/children.js:2]: I think it's fine to leave `replace` as-is, what else could it mean? Not to mention that `replaceChild` might be mistaken for a shallow replace (remember when I was confused about this?).

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+export *                          from "./children.js";
+export {default as closest}       from "./closest.js";
+export {default as evaluate}      from "./evaluate.js";
+export {default as find}          from "../lib/treecle/src/find.js";
+export {default as map}           from "./map.js";
+export {default as serialize}     from "./serialize.js";
+export {default as walk}          from "./walk.js";
+export {default as variables}     from "./variables.js";
+export {default as prepend}       from "./prepend.js";
+export *                          from "./parents.js";
```
[the remark this answers is not in the document]
Comment (LeaVerou) [reply src/index.js:12]: Btw we should fix all these imports as well, there's no reason for the renaming anymore.

**Review by LeaVerou — APPROVED** [review]

File: src/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (adamjanicki2) [reply src/index.js:12]: > Btw we should fix all these imports as well, there's no reason for the renaming anymore.

Do you mean get rid of places where we use `as`? For example, changing the above to `import { update } from "./parents.js"`?

**Pull request merged**